src/main/scala/com/databricks/labs/overwatch/pipeline/Pipeline.scala

```diff
@@ -425,6 +427,33 @@ class Pipeline(
     Helpers.getDatesGlob(startDate.toLocalDate, untilTime.asLocalDateTime.plusDays(1).toLocalDate)
   }
 
+  //Get the dataframe from either tempDirectory or from the ResponseArray
+  private def getApiData(apiTempPath: Option[String]): DataFrame ={
+     //Get the data from either tempDirectory or from the ResponseArray
+     val tmpClusterEventsSuccessPath = s"${config.tempWorkingDir}/${apiTempPath.get}/success" + config.runID
```

Comment:
you're also missing the snapTime here too -- make 10000% sure this path is accurate.

Reply:
i think we can also use the runID in place of snapTime so added the runID.